src/navability/graphql/DataBlobs.jl

```diff
@@ -70,6 +70,50 @@ mutation sdk_adddataentry(\$userId: ResourceId!, \$robotId: ResourceId!, \$sessi
 }
 """
 
+GQL_ADDBLOBENTRY = """
+mutation sdk_addblobentry(
+  \$userId: String!
+  \$robotId: String!
+  \$sessionId: String!
+  \$variableLabel: String!
+  \$blobId: UUID!
+  \$dataLabel: String!
+  \$mimeType: String
+) {
+  addBlobEntry(
+    blob: {
+      id: \$blobId
+      label: \$dataLabel
+      size: 5939
```

Comment:
Note to self: FIXME, hardcoded size slipped in

Reply:
Where would you get the blob size from and why is it a mandatory field?

Reply:
This would be a question for @GearsAD .  I did not have to upload the file size in previous usage, maybe something new or a bug as mandatory?

Reply:
@Affie I'd say in the meantime, just get length of the data on the client and add it to the add call.  My guess is this for hte new large upload file support that Sam is working on.

Reply:
I would just pass ti in `size` as another argument